packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -423,10 +423,10 @@ async function saveExamples(
 ) {
   let examplesToRun = knownExamples.filter(example => example.category === examplesTarget);
 
-  const specificExample = argv.example;
+  const specificExamples = argv.example.split(",").filter(s => s.length > 0);
 
-  if (specificExample) {
-    examplesToRun = examplesToRun.filter(example => example.filename === specificExample);
+  if (specificExamples) {
```

Comment:
@klochek, @Domiii 

I think this is a regression. JavaScript treats an empty string as _false_ in a conditional, but splitting an empty string will always return an empty array, which is _not false_.

This check should be:
```ts
if (specificExamples.length > 0) {
```

Reply:
I filed FE-2168 for this.